x-pack/plugin/src/test/resources/rest-api-spec/api/cat.ml.trained_models.json

```diff
@@ -0,0 +1,100 @@
+{
+  "cat.ml.trained_models":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/get-inference-stats.html"
+    },
+    "stability":"stable",
+    "url":{
+      "paths":[
+        {
+          "path":"/_cat/ml/trained_models",
+          "methods":[
+            "GET"
+          ]
+        },
+        {
+          "path":"/_cat/ml/trained_models/{model_id}",
+          "methods":[
+            "GET"
+          ],
+          "parts":{
+            "model_id":{
+              "type":"string",
+              "description":"The ID of the trained models stats to fetch"
+            }
+          }
+        }
+      ]
+    },
+    "params":{
+      "allow_no_match":{
```

Comment:
cat indices does not have the `allow_no_indices` option. Is this conventional?

Reply:
This matches up with our `GET <resource>` pattern. 

I could remove the option and make it always true.